Source/Particles/ParticleCreation/FilterCreateTransformFromFAB.H

```diff
@@ -58,19 +58,12 @@ Index filterCreateTransformFromFAB (DstTile& dst1, DstTile& dst2, const amrex::B
     const int level_0 = 0;
     Geometry const & geom = warpx.Geom(level_0);
 
-    // Note that we use here x and y for the 2 directions in 2D, which is not consistent with
-    // the rest of the code (x and z are usually chosen)
     const Real xlo_global = geom.ProbLo(0);
-    const Real ylo_global = geom.ProbLo(1);
     const Real dx         = geom.CellSize(0);
-    const Real dy         = geom.CellSize(1);
-#if (AMREX_SPACEDIM == 3)
-    const Real zlo_global = geom.ProbLo(2);
-    const Real dz         = geom.CellSize(2);
-#else
-    const Real zlo_global = 0.0;
-    const Real dz         = 0.0;
-#endif
+    const Real ylo_global = (spacedim == 3) ? geom.ProbLo(1) : amrex:Real(0.);
```

Comment:
The modifications in this file (using z instead of y for the second direction in 2D, as in the rest of the code) are not strictly related to this PR but we were considering doing these changes in #1424 and then I forgot. I figured it would be a good time to do them here.